clio-integration-test/src/it/scala/org/broadinstitute/clio/integrationtest/EnvIntegrationSpec.scala

```diff
@@ -102,10 +94,13 @@ abstract class EnvIntegrationSpec(env: String)
       }, identity)
   }
 
-  override lazy implicit val bearerToken: OAuth2BearerToken = {
-    val credential = serviceAccount.credentialForScopes(authScopes)
-    OAuth2BearerToken(credential.refreshAccessToken().getTokenValue)
-  }
+  override lazy implicit val bearerToken: OAuth2BearerToken =
+    OAuth2BearerToken(
+      AuthUtil
+        .getCredsFromServiceAccount(serviceAccount)
+        .getAccessToken
```

Comment:
~~This used to be `refreshAccessToken()`, did you test the change? I think when I first set this up I got `null` from `getAccessToken` before doing the refresh.~~ Nevermind, I see you call refresh in `getCredsFromServiceAccount`.